### PR TITLE
Setup updates for Wapo implementation

### PR DESCRIPTION
**Pull request by st421**
Just getting a few basics in place that will help us with future development and with producing our own pypi package.

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# This is a comment.
```
Comment (lennybronner) [inline .github/CODEOWNERS:1]: Is it usual to add this kind of stuff to forked code?

File: setup.py [hunk 2]
```diff
@@ -1,29 +1,35 @@
...
-requirements = [
+INSTALL_REQUIRES = (
     'pytz>=2018.4',
-    'requests>=2.19.1',
-    ]
+    'requests<3'
+)
+
+PROJECT = 'wapo-fec'
+AUTHOR = 'The Washington Post Newsroom Engineering Team'
```
Comment (lennybronner) [inline setup.py:13]: similarly here

File: .github/CODEOWNERS (hunk 1 above)
Comment (st421) [reply .github/CODEOWNERS:1]: First time doing this so not sure what is typical, but my thought process with this PR was:

We will be maintaining this version of the package. We need folks from our team to review PRs. If someone uses the pypi package we publish and finds and issue, they should contact us about it (it is possible the issue will apply to both versions of the package, but we certainly don't want anyone reaching out to the original author about issues that are specific to our fork, right?). So I think these updates are probably good/right, but we should make sure to clearly credit the original author in other locations.

Comment (freedmand) [reply .github/CODEOWNERS:1]: This seems OK to me

File: setup.py (hunk 2 above)
Comment (freedmand) [reply setup.py:13]: We would be the author of the PIP package. I think we should be explicit in the README that this is forked code

Comment (freedmand) [reply setup.py:13]: Approve now given README changes

**Review by freedmand — DISMISSED** [review]
Seems reasonable all around

**Review by lennybronner — APPROVED** [review]

**Pull request merged**